modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -173,8 +173,8 @@ public Organization postOrganization(Organization organization)
 				ServiceContextFactory.getInstance(contextHttpServletRequest));
 
 		return _organizationResourceDTOConverter.toDTO(
-			_getDTOConverterContext(String.valueOf(serviceBuilderOrganization)),
-			serviceBuilderOrganization);
+			_getDTOConverterContext(
+				serviceBuilderOrganization.getExternalReferenceCode()));
```

Comment:
The result of `getExternalReferenceCode` will always be `null` here. We should use `String.valueOf(serviceBuilderOrganization.getOrganizationId())` instead.

Reply:
I see. Thanks for catching that, @drewbrokke! Would you like us to submit a new pull with the revision?

Reply:
Yes please :)

Reply:
Sounds good! We will do that.